controllers/menuDiscountController.js

```diff
@@ -82,6 +82,32 @@ const getMenuDiscountsByMerchant = async (req, res) => {
     }
 };
 
+const getMenuDiscountsByMerchantAndDiscount = async (req, res) => {
+    try {
+        const { merchantId, discountId } = req.params;
+
```

Comment:
@hchenghy You have to check first if  merchantiId and discounted are existing. If not, send a status 4xx and a message.